controller/api/destination/profile_translator.go

```diff
@@ -90,9 +99,10 @@ func toServiceProfile(profile *sp.ServiceProfile) (*pb.DestinationProfile, error
 		budget.Ttl = toDuration(ttl)
 	}
 	return &pb.DestinationProfile{
-		Routes:       routes,
-		RetryBudget:  budget,
-		DstOverrides: toDstOverrides(profile.Spec.DstOverrides),
+		Routes:             routes,
+		RetryBudget:        budget,
+		DstOverrides:       toDstOverrides(profile.Spec.DstOverrides),
+		FullyQualifiedName: profile.Name,
```

Comment:
Thinking about this some more, since we already have the FQN on the translator anyway, we might as well just use that.  It should be the same as the ServiceProfile name but this way the logic is more consistent between the default and non-default cases.  It also means we don't need to craft a name for the merged service profile in the traffic split adaptor.

Doing this will require `toServiceProfile` to be a method on `profileTranslator` but that should be fine since it's only called from `Update`.

Reply:
Yea sounds good I think keeping it consistent is better